Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
As per the rest of the answer, I think I need to read some code before I can understand it, I'll try to do that soon.